build.sbt

```diff
@@ -139,12 +139,23 @@ lazy val node = (project in file("node"))
     maintainer in Linux := "Pyrofex, Inc. <info@pyrofex.net>",
     packageSummary in Linux := "RChain Node",
     packageDescription in Linux := "RChain Node - the RChain blockchain node server software.",
+     linuxPackageMappings += {
+      val file = baseDirectory.value / "rnode.service"
+      packageMapping( (file -> "/lib/systemd/system/rnode.service") )
+    },
     /* Debian */
+    packageSummary in Debian := "RChain Node",
+    packageDescription in Debian := "RChain Node - the RChain blockchain node server software.",
```

Comment:
These two lines are redundant - remove them.  This settings should be inherited from `Linux`.